### DISCUSSION TITLE
Can't get verified for the Student Pack

### DISCUSSION THREAD

**Discussion by aaroncherian** [community/community#55677] [GitHub Education] [answered]
### Hi everyone,

Hey there,

Sort of at the end of my rope here, but I've submitted multiple times to be verified as a student and they've all been rejected - at this point, I've uploaded a picture of my student ID, my proof of enrollment, and a page from my academic transcript. I tried to put all the pages of my academic transcript together and submit it, but the photo was marked as 'edited'. I then just took the first page of the transcript, up-res'd it (it was apparently too small to submit), and then it was still marked as being 'edited' or manipulated.

Truly not sure what else to do here, and help would be appreciated.

**Comment by jge162** [comment, marked as answer]
If you are using your school email on your GitHub account and upload a page of the transcript showing your current term of enrollment and has school name and your name that should work. That is how I got verified. Good luck. @aaroncherian

**Reply by aaroncherian** [reply]
Hey thanks - gave it another shot with my ID after changing the email on my github account and it seems like it was approved. Very much appreciate the assistance!

**Reply by jge162** [reply]
Nice enjoy all the features they are amazing...Welcome.
